smac/runhistory/runhistory.py

```diff
@@ -314,7 +323,7 @@ def update_from_json(self, fn, cs):
         """
         new_runhistory = RunHistory(self.aggregate_func)
         new_runhistory.load_json(fn, cs)
-        self.update(runhistory=new_runhistory)
+        self.update(runhistory=new_runhistory, external_data=True)
```

Comment:
Thinking more about it, I even believe that treating loaded runhistories as external in the context of pSMAC is wrong. external_data is not visible in the intensification procedure but we want to have it visible there.